src/search-exclude.php

```diff
@@ -132,6 +133,14 @@ protected function isExcluded($postId)
         return false !== array_search($postId, $this->getExcluded());
     }
 
+    public function excludeFromSearchEngines()
+    {
+        global $post;
+        if (false !== array_search($post->ID, get_option('sep_exclude', array())) && get_option('sep_exclude_from_search_engines')) {
```

Comment:
Since this is a plugin context, you could use `$this->isExcluded($post->ID)` instead of `false !== array_search($post->ID, get_option('sep_exclude', array()))`